src/flags.ts

```diff
@@ -42,7 +42,7 @@ export const version = (opts: Partial<BooleanFlag<boolean>> = {}) => {
 
 export const help = (opts: Partial<BooleanFlag<boolean>> = {}) => {
   return Parser.flags.boolean({
-    // char: 'h',
+    char: 'h',
```

Comment:
Do we still want this change since it's now configurable?

Reply:
Do not need this change, will revert.